### PR TITLE
README: Fix a vectors section typo

### PR DESCRIPTION
**Pull request by sameo**
And improve the prose a little bit as well.

Signed-off-by: Samuel Ortiz <sameo@rivosinc.com>

### CODE REVIEW

**Comment by sameo** [comment]
Nitty nit...

**Comment by dgreid** [comment]
applied with one edit s/processor/processors/

File: README.md [hunk 1]
```diff
@@ -177,6 +177,6 @@ commandline.
 
 ### Vectors
 
-Salus is able to detect if the CPU has the vector extensio. The same binary will 
-run on processor with or without the vector exension, and will enable vector code
-if the extension is present.
+Salus is able to detect if the CPU supports the vector extension. The same
+binary will run on processor with or without the exension, and will enable
```
Comment (dgreid) [inline README.md:181]: nit s/processor/processors.

**Review by dgreid — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (sameo) [reply README.md]: Done.

**Pull request merged**